### PR TITLE
Bug fixes faruq

### PR DESCRIPTION
**Pull request by muhammad-faruq**

### CODE REVIEW

File: src/main/java/seedu/anilist/ui/StatsDisplay.java [hunk 1]
```diff
@@ -140,8 +140,13 @@ public void setAnimeListStats(Stats stats) {
 
     private void setAnimeStats(Stats stats) {
         pieChart.getData().clear();
-        pieChart.setTitle(String.format(TOTAL_ANIMES_MSG, stats.getTotalAnimesCount())
-                + "\n" + String.format(EPISODES_WATCHED_MSG, stats.getEpisodesCount()));
+        if (stats.getTotalAnimesCount() == 0) {
+            pieChart.setId("pieChartEmpty");
```
Comment (auhc99) [inline src/main/java/seedu/anilist/ui/StatsDisplay.java:144]: Use a constant here?

File: src/main/java/seedu/anilist/ui/StatsDisplay.java [hunk 2]
```diff
@@ -140,8 +140,13 @@ public void setAnimeListStats(Stats stats) {
...
+            pieChart.setTitle(NO_ANIMES_MSG);
+            return;
+        }
+        pieChart.setId("pieChart");
```
Comment (auhc99) [inline src/main/java/seedu/anilist/ui/StatsDisplay.java:148]: same

File: src/main/java/seedu/anilist/ui/StatsDisplay.java [hunk 3]
```diff
@@ -168,6 +172,15 @@ public String toString(Number object) {
         barChart.setData(tempGenreStats.getData());
         barChart.getYAxis().setTickMarkVisible(false);
         barChart.getXAxis().setTickMarkVisible(false);
+        if (genreStats.isEmpty()) {
+            barChart.setId("barChartEmpty");
```
Comment (auhc99) [inline src/main/java/seedu/anilist/ui/StatsDisplay.java:176]: same

File: src/main/java/seedu/anilist/ui/StatsDisplay.java [hunk 4]
```diff
@@ -168,6 +172,15 @@ public String toString(Number object) {
...
+            barChart.getXAxis().setOpacity(0);
+            barChart.setTitle(NO_GENRES_TO_SHOW_MSG);
+        } else {
+            barChart.setId("barChart");
```
Comment (auhc99) [inline src/main/java/seedu/anilist/ui/StatsDisplay.java:180]: same

**Review by auhc99 — CHANGES_REQUESTED** [review]
Use constants

File: src/main/java/seedu/anilist/ui/StatsDisplay.java (hunk 4 above)
Comment (luodan01) [reply src/main/java/seedu/anilist/ui/StatsDisplay.java:180]: ok

**Review by auhc99 — APPROVED** [review]
LGTM

**Pull request merged**